### ISSUE TITLE
xml multi-line comments in code block produce garbage

### ISSUE THREAD

**Issue by bugariu** [doxygen/doxygen#9522]
**Describe the bug**
xml multi-line comments in code block produce garbage

**Expected behavior**
Comment should be visible

**Screenshots**
![garbage](https://user-images.githubusercontent.com/5676532/185072437-6c42be8c-fbad-4bd3-9f53-3cc865d311b5.png)

**To Reproduce**
```
/**
 * @brief This is an example base class.
 *
 * It contains a secret number and provides an abstract method to do something.
 * @code{.xml}
 * gnagna
 * <!-- B3/4/5 indicators
 * -->
 * <a>foo</a>
 * @endcode
 *
 * @note        This is a note.
 * @attention   Attention please!
 * @warning     This is a warning.
 * @bug         This is a bug. Please fix it!
 */

```

**Version**
1.9.1

**Stack trace**
none

**Additional context**
none

**Comment by albert-github** [comment]
With the 1.9.1 version of doxygen I can reproduce the problem.

With the current doxygen version 1.9.4 all looks OK:

![image](https://user-images.githubusercontent.com/5223533/185074278-f85e966c-3057-4433-a08b-3dc8a78c8509.png)


I think the problem has been solved in the mean time.

**Comment by bugariu** [comment]
Cool!

**Comment by bugariu** [comment]
It seems that it was fixed by #8322  
IMO this ticket can be closed.

**Comment by bugariu** [comment]
Duplicate of #8322
